website/docs/core-concepts/connecting-to-data-sources/authentication/connect-to-apis.md

```diff
@@ -4,24 +4,168 @@ description: You can hit any REST endpoint available on the public internet
 
 # Configuring APIs
 
+Configuring APIs involves setting up and managing the various aspects of an API that allow it to function effectively. This can include defining the API's structure and capability, setting up authentication and authorization, and establishing connections with other systems or services.
+
 :::tip
-If your API is not available on the public internet, you must expose it via a service such as [https://ngrok.com/](https://ngrok.com/)
+If your API isn't available on the public internet, you must expose it via a service such as [https://ngrok.com/](https://ngrok.com/)
```

Comment:
The cross-reference should be to the local DB and API page for using ngrok.

Reply:
This is still open. Please add reference to - /advanced-concepts/more/how-to-work-with-local-apis-on-appsmith#using-hostdockerinternal